app/controllers/comments_controller.rb

```diff
@@ -1,2 +1,18 @@
 class CommentsController < ApplicationController
+  def create
+    comment = Comment.new(comment_params)
+    respond_to do |r|
+      r.html do
+        comment.save
+        redirect_to user_posts_path(current_user)
+      end
+    end
+  end
+
+  def comment_params
+    params
+      .require(:comment)
+      .permit(:text)
+      .merge(author: current_user, post_id: params.require(:post_id))
+  end
```

Comment:
- [ ] [OPTIONAL] This method should be private, kindly add a ```private``` before the method
e.g
```ruby 
private

def comment_params
...
end
```